timesketch/ui/graphs/cytoscape.component.spec.ts

```diff
@@ -0,0 +1,264 @@
+import {Component, ViewChild} from '@angular/core'
+import {TestBed, ComponentFixture} from '@angular/core/testing'
+import {BrowserDynamicTestingModule, platformBrowserDynamicTesting} from '@angular/platform-browser-dynamic/testing'
+import {BrowserModule} from '@angular/platform-browser'
+
+import {CytoscapeComponent} from './cytoscape.component'
+
+export const testCytoscapeOptions = {
+  zoomingEnabled: false,
+  userZoomingEnabled: false,
+  panningEnabled: false,
+  userPanningEnabled: false,
+  boxSelectionEnabled: true,
+  autolock: true,
+  autoungrabify: true,
+  autounselectify: true,
+  headless: true,
+  hideEdgesOnViewport: true,
+  hideLabelsOnViewport: true,
+  textureOnViewport: true,
+  motionBlur: true,
+  styleEnabled: false,
+  minZoom: 1e-10,
+  maxZoom: 1e10,
+  touchTapThreshold: 10,
+  desktopTapThreshold: 6,
+  motionBlurOpacity: 0.4,
+  wheelSensitivity: 0.8,
+  pixelRatio: 1.0,
+  selectionType: 'additive',
+}
+
+export const getOptionFromCytoscape = (_cy: Cy.Core, option: string) => {
+  const cy = _cy as any
```

Comment:
Can we have a better type for cy here, e.g. use a intersection type?

Reply:
This is hacky code only for use in tests, it accesses cytoscape internals, I don't see point in adding accurate type annotations here. Correctness of this code is already checked when running unit tests, so type errors will not add anything.

Reply:
Done. There is an intersection type for `cy`.